frontend/src/components/Selectors/FormatSelector.tsx

```diff
@@ -0,0 +1,24 @@
+import { chain } from 'lodash'
+import { DropDownMenu } from 'material-ui/DropDownMenu'
+import MenuItem from 'material-ui/MenuItem'
+import * as React from 'react'
+import { Format } from '../../constants/datatypes'
+
+type Props = {
+  onChange(format: Format): void
+  value: Format
+}
+
+let options: Format[] = ['Percentage', 'Number']
+let menuItems = chain(options).map(
+  _ => <MenuItem value={_} key={_} primaryText={_} />
+).value()
+
+export let FormatSelector: React.StatelessComponent<Props> = ({ onChange, value }) =>
```

Comment:
Would be nice to have this as a toggle switch instead, but maybe that should wait for a future PR?

Reply:
This is the current UI request, butmaybe something to think about @viviandien ?